app/dashboard/page.tsx

```
@@ -0,0 +1,1116 @@
+                                  'prose-headings:dark:text-cyan-200',
+                                  'prose-p:dark:text-gray-300',
+                                  'prose-strong:dark:text-cyan-300',
+                                  'prose-code:dark:text-pink-400 prose-code:dark:bg-gray-800',
+                                  'prose-pre:dark:bg-gray-900 prose-pre:dark:border prose-pre:dark:border-cyan-400/20',
+                                  'prose-blockquote:dark:border-cyan-400/50 prose-blockquote:dark:text-gray-400',
+                                  'prose-li:dark:text-gray-300',
+                                  'prose-a:text-cyan-500 prose-a:hover:text-cyan-400'
+                                )}
+                                dangerouslySetInnerHTML={{
+                                  __html: tiptapJSONtoHTML(
+                                    currentEditingPost.content
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)